workers/tasks/export/metadata.js

```diff
@@ -50,6 +50,7 @@ export const getPubMetadata = async (pubId) => {
 		publishedDateString: publishedDateString,
 		updatedDateString: updatedDateString,
 		communityTitle: pubData.community.title,
+		communityAccentColorDark: pubData.community.accentColorDark,
```

Comment:
My weak suggestion here is that you could call this field `accentColor` or even `color`, to decouple the rendering code from the idea of where specifically the color came from.

Reply:
👍 
Going with `accentColor` because I can see other colors being added in the future.